app/models/pawn.rb

```diff
@@ -7,4 +7,54 @@ def unicode_symbol
       return "&#9823;"
     end
   end
+
+  def valid_move?(destination_x, destination_y)
+    valid = super(destination_x, destination_y)
+    if valid 
+      if pawn_diagonal?(destination_x, destination_y)
+        capture_piece(destination_x, destination_y)
+        return true
+      elsif move_backward?(destination_y) && (one_square?(destination_x, destination_y) || two_squares?(destination_x, destination_y))
+        return false
+      end
+      move_backward?(destination_y) 
+    end
+    return valid
+  end
+
+  def on_home_row?
+    return true if (self.get_color == WHITE && y_position == 1) || y_position == 6
```

Comment:
Yep, thanks @kellylougheed - I added it to the line to take care of an ambitious WHITE pawn.